### PR TITLE
Feature/change autoprefixer

### PR DESCRIPTION
**Pull request by kkateivanova**
Use postcss-loader instead of autoprefixer-loader as recommended https://github.com/passy/autoprefixer-loader.

### CODE REVIEW

File: generators/app/templates/app/scripts/app.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const sblocksComponents = [
   // simple-blocks implementation
   // https://github.com/lexich/simple-blocks
 ];
-const Modernizr = window.Modernizr;
+// const Modernizr = window.Modernizr;
```
Comment (maximkoretskiy) [inline generators/app/templates/app/scripts/app.js]: We should remove this line

File: generators/app/templates/webpack/config.coffee [hunk 2]
```diff
@@ -79,6 +80,7 @@ doConfig = ->
       /jquery\/dist\/jquery\.js/
       /^backbone\/backbone\.js/
     ]
+  postcss: [ autoprefixer({ browsers: ['last 2 versions'] }) ]
```
Comment (maximkoretskiy) [inline generators/app/templates/webpack/config.coffee]: Use multiline array syntax to give more convenient way to plugins add

File: generators/app/templates/app/scripts/app.js (hunk 1 above)
Comment (kkateivanova) [inline generators/app/templates/app/scripts/app.js]: I think this show optional adding of modernizr using, like in webpack settings - if you want to use - uncomment.

File: generators/app/templates/webpack/config.coffee [hunk 3]
```diff
@@ -3,6 +3,7 @@ requireChild = require '../gulp/requireChild'
 webpack = requireChild('webpack')
 path = require 'path'
 ExtractTextPlugin = require 'extract-text-webpack-plugin'
+autoprefixer = require 'autoprefixer'
```
Comment (maximkoretskiy) [inline generators/app/templates/webpack/config.coffee]: Let's use inline require for postcss plugins
```coffee
postcss: [
  (require 'autoprefixer') { browsers: ['last 2 versions'] }
]
```

File: generators/app/templates/app/scripts/app.js (hunk 1 above)
Comment (maximkoretskiy) [inline generators/app/templates/app/scripts/app.js]: Modernizr is available globally. This line means nothing. Write usual comment to inform developer.

**Pull request merged**